app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors."""
+    pass

```

Comment:
Consider adding a descriptive error message to the `VaccineError` exception. For example: `class VaccineError(Exception): def __init__(self, message='Vaccine error occurred'): super().__init__(message)`.